src/tenant/tenant.service.ts

```
@@ -0,0 +1,26 @@
+    return 'This action adds a new tenant';
+  }
+
+  findAll() {
+    return `This action returns all tenant`;
+  }
+
+  findOne(id: number) {
+    return `This action returns a #${id} tenant`;
+  }
+
+  update(id: number, updateTenantDto: UpdateTenantDto) {
```

ESLint

[error] 'updateTenantDto' is defined but never used.

(@typescript-eslint/no-unused-vars)